readme.md

```diff
@@ -124,6 +124,72 @@ Also, you may be interested in adjusting Nextcloud's Datadir to store the files
 
 ⚠️ **Please note:** Almost all commands in this project's documentation use `sudo docker ...`. Since `sudo` is not available on Windows, you simply remove `sudo` from the commands and they should work.  
 
+
+### Synology DSM Setup
+<details>
+    <summary>For a AIO setup in Docker on a Synology NAS, there are a few adjustments necessary to make it run well, especially when running the integrated reverse proxy or firewall.</summary>
+    
+    
+The main issue is that installing AIO through the **Docker GUI doesn't work**, so you need to either use SSH or create a user-defined script task in the task scheduler as the user 'root' and run it.
+Before you start make sure to create two folders on you NAS `/volume1/docker/nextcloud/data` `/volume1/docker/nextcloud/backup`
+
+The compose script without reverse proxy
+```
+sudo docker run \
+--sig-proxy=false \
+--name nextcloud-aio-mastercontainer \
+--restart always \
+--publish 80:80 \
+--publish 8080:8080 \
+--publish 8443:8443 \
+--volume nextcloud_aio_mastercontainer:/mnt/docker-aio-config \
+--volume /volume1/docker/docker.sock:/var/run/docker.sock:ro \
+-e NEXTCLOUD_DATADIR="/volume1/docker/nextcloud/data" \
```

Comment:
I'd say lets not add this here as it is an optional thing. Howevwer lets link to the correct section below the command (like we did with the windows install)

```suggestion
```

Reply:
Ok I didn't know that this was optional.

But maybe add a note to let people know that there is more on Synology further below?

Reply:
See https://github.com/nextcloud/all-in-one/pull/1813/files#r1085140153